diesel/src/types/mod.rs

```diff
@@ -292,12 +316,10 @@ pub struct Date;
 ///
 /// [`PgInterval`]: ../pg/data_types/struct.PgInterval.html
 /// [`IntervalDsl`]: ../pg/expression/extensions/trait.IntervalDsl.html
-#[derive(Debug, Clone, Copy, Default, QueryId)]
+#[derive(Debug, Clone, Copy, Default, QueryId, SqlType)]
+#[postgres(oid = "1186", array_oid = "1187")]
 pub struct Interval;
 
-#[cfg(not(feature = "postgres"))]
```

Comment:
The new derive adds this impl also for Postgres. Is this intended?

Reply:
The impl was always present it just came from a different place if PG was enabled.